validator/src/main.rs

```diff
@@ -1413,20 +1420,10 @@ pub fn main() {
     validator_config.account_snapshot_paths = account_snapshot_paths;
 
     validator_config.account_shrink_paths = account_shrink_paths.map(|paths| {
-        paths
```

Comment:
we were doing the same thing in 2 places, just moved it to a function so we can also use this in ledger-tool when I get to checking that out.